themes/bootstrap3/js/common.js

```diff
@@ -706,6 +707,7 @@ $(function searchFormResetHandler() {
   });
   resetButton.on('click', function onClick() {
     queryInput.attr('value', '').focus();
+    queryInput[0].ac.hide(); // not sure why AC is using previous value
```

Comment:
That's because autocomplete listens to the input event, and clearing the contents programmatically doesn't trigger the event. Something like this should help:

`queryInput.attr('value', '').trigger('focus')[0].dispatchEvent(new Event('input'));`

(`focus()` is deprecated, that's why `trigger('focus')` instead). This will still display the "Loading..." message, and I'm not sure if that's correct, but it's the same with the initial keystrokes so not in scope here.

Reply:
This definitely answers my question, but I think the hide method is better since it doesn't flash Loading or anything else.

Reply:
I'm fine with that, but perhaps autocomplete could also listen for the `change` event to reset when a field is emptied? I think it would be better to not leave the old state dangling regardless of whether it shows.

Reply:
True. I’ll add a “change” listener and see if it fixes this issue without doubling up on what “input” is already doing.